From: Boyuan Yang <073plan@gmail.com>
Date: Wed, 13 Dec 2017 16:03:36 +0800
Subject: Patch to search system Qt private headers.

This patch would let qmake search for system qt headers instead
of using bundled headers.
---
 platformplugin/linux.pri | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/platformplugin/linux.pri b/platformplugin/linux.pri
index a6899a7..72eed94 100644
--- a/platformplugin/linux.pri
+++ b/platformplugin/linux.pri
@@ -65,6 +65,8 @@ contains(QT_CONFIG, xcb-qt) {
 
 exists($$PWD/libqt5xcbqpa-dev/$$VERSION) {
     INCLUDEPATH += $$PWD/libqt5xcbqpa-dev/$$VERSION
+} else:exists($$[QT_INSTALL_HEADERS]/QtXcb/$$[QT_VERSION]) {
+    INCLUDEPATH += $$[QT_INSTALL_HEADERS]/QtXcb/$$[QT_VERSION]/QtXcb/private
 } else {
     error(Not support Qt Version: $$VERSION)
 }
